### DISCUSSION TITLE
Overflow scroll on popup is not working

### DISCUSSION THREAD

**Discussion by Mediaploeg** [darkroomengineering/lenis#242] [Q&A] [answered]
Hi guys, I use lenis smooth scrolling on my Webflow website. On my website I have a popup which opens after clicking on a button. The popup is bigger than the viewport, so you need to be able to scroll. But when I open the pop up I'm not able to scroll in the pop up.

Any ideas how to fix this?

The website page I'm talking about is: https://tcxa.webflow.io/contact

The pop up opens after you click on the button "start a project"

**Comment by muravevnv** [comment, marked as answer]
you need add data attribute data-lenis-prevent to block-wrapper of your popup
example `<div class="some-class" data-lenis-prevent></div>`

**Reply by erwin-yndenz** [reply]
Thanks! I've tried so many different google searches, but google just would not let me find how to solve it! You saved the day

PS: I see it in the docs now, but it was ambigious to me, not even some explanation of when to use it

**Reply by kipedia** [reply]
I'm doing something similar on my site, scrolling of overflow on the modal works for me, but it doesn't keep the smooth-scroll active, did you find a way to resolve this?

**Reply by GamendeBrian** [reply]
On calling `lenis.stop()`, the overflow issue inside a modal occured. This fixed it. Thanks!
